### PR TITLE
docs: expand troubleshooting section on clock drift

### PR DESCRIPTION
**Pull request by milos-teleport**
Duplicate/alternative-to #46487 

We have another case of system-clock related issue which happens if the auth server instance is out of sync.
@zmb3 suggested we bundle the previous Windows system clock issue together with this one.

### CODE REVIEW

File: docs/pages/enroll-resources/desktop-access/troubleshooting.mdx [hunk 1]
```diff
@@ -354,21 +354,49 @@ connections.
...
+           Universal time: Wed 2024-09-11 13:45:17 UTC
+                 RTC time: Wed 2024-09-11 13:45:17
+                Time zone: Etc/UTC (UTC, +0000)
+System clock synchronized: yes
+              NTP service: active
+          RTC in local TZ: no
+```
+
+If `System clock synchronized` is `no`, this would need to be resolved.
+This should also be checked on your Teleport proxies and Teleport agents to get ahead of other issues which relate to system clock drift.
```
Comment (milos-teleport) [inline docs/pages/enroll-resources/desktop-access/troubleshooting.mdx:398]: Potential scope-creep. I wonder how people feel about this one

Comment (ptgott) [inline docs/pages/enroll-resources/desktop-access/troubleshooting.mdx:398]:
```suggestion
This should also be checked on your Teleport Proxy Service instances and Teleport Agents to get ahead of other issues which relate to system clock drift.
```
For standard Teleport component naming in the docs

**Review by ptgott — APPROVED** [review]

**Review by mmcallister — APPROVED** [review]

**Comment by zmb3** [comment]
@mmcallister @ptgott I've moved things around and reworded this a bit with Milos approval, so I'd appreciate if you took another review pass 🙏

File: docs/pages/enroll-resources/desktop-access/troubleshooting.mdx [hunk 2]
```diff
@@ -5,6 +5,63 @@ description: Common issues and resolutions for Teleport's desktop access
...
+have an incorrect system clock.
+
+For example, you may see that Teleport starts a remote desktop session, but that
+the login screen on the remote desktop shows an error similar to:
+
+```text
+The smartcard certificate used for authentication has expired. Please contact your system administrator.
+```
+
+Alternatively, you may see an error in the Teleport web UI before the remote desktop connection
```
Comment (mmcallister) [inline docs/pages/enroll-resources/desktop-access/troubleshooting.mdx:25]:
```suggestion
Alternatively, you may see an error in the Teleport Web UI before the remote desktop connection
```

**Comment by mmcallister** [comment]
> @mmcallister @ptgott I've moved things around and reworded this a bit with Milos approval, so I'd appreciate if you took another review pass 🙏

lgtm!

**Review by ptgott — APPROVED** [review]

**Pull request merged**